W-10731885 Google BigQuery 1.0

sathya0 opened the pull request.

sathya0 committed:
interim

sathya0 committed:
more

sathya0 committed:
Update google-bigquery-connector-reference.adoc

sathya0 committed:
Update google-bigquery-connector-reference.adoc

sathya0 committed:
more

sathya0 committed:
more

sathya0 committed:
more

sathya0 committed:
more

sathya0 committed:
Update google-bigquery-connector-examples.adoc

sathya0 committed:
edits

sathya0 committed:
Update google-bigquery-connector-reference.adoc

sathya0 committed:
updated ref

sathya0 committed:
example changes

sathya0 committed:
imgs

sathya0 committed:
more

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+[%header,cols="30s,70a"]
+|===
+|Field |Value
+|Project Id | `attributes.queryParams.projectId`
```
Remove projectId since it is no longer a parameter for operations

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+[%header,cols="30s,70a"]
+|===
+|Field |Value
+|Project Id | `attributes.queryParams.projectId`
```
Remove projectId since it is no longer a parameter for operations

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+[%header,cols="30s,70a"]
+|===
+|Field |Value
+|Project Id | `attributes.queryParams.projectId`
```
Remove projectId since it is no longer a parameter for operations

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+[%header,cols="30s,70a"]
+|===
+|Field |Value
+|Project Id | `attributes.queryParams.projectId`
```
Remove projectId since it is no longer a parameter for operations

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+. From a web browser, test the application by entering `+http://{app-name}.cloudhub.io+`.
+
+[[get-job-result]]
+== Get a Job Result
```
```suggestion
== Get a Query Result
```

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+. Click the *Authorize* button for OAuth2 authorization.
+. From a web browser, test the application by entering `+http://{app-name}.cloudhub.io+`.
+
+[[get-job-result]]
```
```suggestion
[[get-query-result]]
```

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+* <<get-job>>
+* <<insert-job>>
+* <<delete-job>>
+* <<get-job-result>>
```
```suggestion
* <<get-query-result>>
```
I found out there are references to **Get Query Result** as **Get Job Result**, I will be updating this into our demos/readme/etc to match the suggestions here

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+* *Logger* +
+Shows the HTTP response from the *Get Query Results* operation
+
+image::get-job-result.png[Studio Flow for the Get Query Results operation]
```
I think we should rename this file to get-query-result

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+[%header,cols="30s,70a"]
+|===
+|Field |Value
+|Project Id | `attributes.queryParams.projectId`
```
Remove projectId since it is no longer a parameter for operations

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+[%header,cols="30s,70a"]
+|===
+|Field |Value
+|Project Id | `attributes.queryParams.projectId`
```
Remove projectId since it is no longer a parameter for operations

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+[%header,cols="30s,70a"]
+|===
+|Field |Value
+|Project Id | `attributes.queryParams.projectId`
```
Remove projectId since it is no longer a parameter for operations

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+					authorizePath="/authorize" />
+		</bigquery:oauth2-connection>
+	</bigquery:config>
+	<flow name="getJobResultFlow">
```
```suggestion
	<flow name="getJobQueryFlow">
```
I will update these suggestions in our code as well

tomastoloza commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc:
```diff
@@ -0,0 +1,1182 @@
+	<flow name="getJobResultFlow">
+		<http:listener doc:name="Listener"
+					   config-ref="HTTP_Listener_config" path="/get/results" />
+		<bigquery:get-query-results doc:name="Get Job Result Paged" config-ref="BigQuery__Configuration" maxResults="25" jobId="#[attributes.queryParams.jobId]">
```
```suggestion
		<bigquery:get-query-results doc:name="Get Query Result Paged" config-ref="BigQuery__Configuration" maxResults="25" jobId="#[attributes.queryParams.jobId]">
```

tomastoloza reviewed (changes requested):
Some comments regarding operation names.
I will be updating the suggestions in our code as well to avoid any discrepancies.

sathya0 committed:
tech edits

tomastoloza reviewed (dismissed):
Looks good, thank you!

javiermule committed:
W-10731885 Adding access_type field explanation

Added explanation extracted from https://developers.google.com/identity/protocols/oauth2/web-server#httprest_1 . @sathya0

sathya0 committed:
edit

sathya0 committed:
Update google-bigquery-connector-reference.adoc

sathya0 committed:
Merge pull request #1853 from mulesoft/W-10731885-google-bigquery-1.0-OAuth-details

W-10731885 Adding access_type field explanation

tasmoore commented on google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-studio.adoc:
```diff
@@ -0,0 +1,160 @@
+
+image::oauth2-auth.png[*OAuth2* is selected in the *Connection* section and authentication fields are completed in the *General* tab.]
+
+The first item shows where to specify that the connector uses OAuth2 authentication. The second item shows the *General tab*, which contains fields related to OAuth2 authentication.
```
Should "tab" be unbolded here?

sathya0 committed:
Update google-bigquery-connector-studio.adoc

sathya0 committed:
Merge branch 'latest' into W-10731885-google-bigquery-1.0-sg
